.gitlab-ci.yml

```diff
@@ -375,7 +376,7 @@ fuzz-tests:
     <<:                              *docker-env
     variables:
         # The QUICKCHECK_TESTS default is 100
-        QUICKCHECK_TESTS:            40000
+        QUICKCHECK_TESTS:            20000
```

Comment:
Is there a way we could define a timeout instead?

Reply:
So the GitLab CI run times out after one hour, that's a hard timeout. But quickcheck only has this level of control.